### PR TITLE
🐛 Source Amazon Seller Partner: attempt to fix an issue with role_arn required with authorization is done via oauth

### PR DESCRIPTION
**Pull request by andreyAtBB**
## What
When user creates Amazon Seller Partner connector source on Airbyte Cloud, authenticate via Oauth and save and error appears as described in these issues:
https://github.com/airbytehq/airbyte/issues/28727
https://github.com/airbytehq/airbyte/issues/27495

## How
If role_arn is not provided (which is the case for Oauth authentication) do not attempt to split it and simply proceed.

## Recommended reading order
n/a

## 🚨 User Impact 🚨
Should not have any impact but should allow user to proceed one step further during connector set up.

## Pre-merge Actions
None

### CODE REVIEW

**Comment by rileybrook** [comment]
@lazebnyi Hoping this small PR can be shepherded through as it blocks an Airbtye Cloud customer from onboarding new clients.

File: airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml:5]: Can you bump also the doc changelog?
After that I think it's fine to merge. Right now the `role arn` is optional and if you don't specify it will break.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml
Comment (andreyAtBB) [reply airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml:5]: done. could you take a look.

**Review by marcosmarxm — APPROVED** [review]

**Comment by andreyAtBB** [comment]
@marcosmarxm, there are some workflows awaiting for approval from a maintainer blocking the ability to merge the fix in.
Is this something you can look at?

**Review by lazebnyi — APPROVED** [review]

**Comment by marcosmarxm** [comment]
@andreyAtBB I'm running tests right now to release the change.

**Pull request merged**
